fragments/shared/validation/success_on_first_boot.ks

```diff
@@ -0,0 +1,59 @@
+# Report success if no errors have been reported on the first boot.
```

Comment:
Please add another header comment that the file `/usr/libexec/kickstart-test.sh` should be created by the test.

Reply:
Fixed.